extensions/common/node_info.fbs

```diff
@@ -0,0 +1,20 @@
+namespace Wasm.Common;
```

Comment:
Can this file generated from `flatc`? It can take `.proto` as input and covert to `fbs`.

Reply:
I need the key attribute. I can try it but it's a bit more opaque to through proto.

Reply:
flatc doesn't seem to support protobuf map. I think we can just delete proto version after the transition is doe.